src/jit/codegenlinear.h

```diff
@@ -173,6 +173,14 @@ void genCodeForShiftLong(GenTreePtr tree);
 void genCodeForShiftRMW(GenTreeStoreInd* storeInd);
 #endif // _TARGET_XARCH_
 
+void genCodeForCast(GenTreeOp* tree);
+
```

Comment:
Do we need all these empty lines between declarations? They don't seem to contribute to readability.